src/w3d/renderer/inttest.h

```diff
@@ -66,6 +66,7 @@ inline bool AABoxIntersectionTestClass::Cull(const Vector3 &cull_min, const Vect
     if ((box_min.Z > cull_max.Z) || (box_max.Z < cull_min.Z)) {
         return true;
     }
+    return false;

```

Comment:
Makes me wonder how did this ever compile?

Reply:
int and bool functions do compile without return statements, but should atleast emit a warning. It's UB however

Reply:
would love if we could make this a error not a warning somehow

Reply:
for these two likely need to check what code uses them and how it can impact the game

Reply:
Ok but original was using old MSVC, so how was it in original then?

Here is list of disabled MSVC warnings.
https://learn.microsoft.com/en-us/cpp/preprocessor/compiler-warnings-that-are-off-by-default?view=msvc-170

I did not spot anything regarding function return statements.